src/Application/PresenterFactory.php

```diff
@@ -35,11 +35,15 @@ class PresenterFactory extends Nette\Object implements IPresenterFactory
 	/** @var bool */
 	private $autoRebuild;
 
+	/** @var int */
+	private $invalidLinkMode;
 
-	public function __construct(Nette\DI\Container $container, $autoRebuild = FALSE)
+
+	public function __construct(Nette\DI\Container $container, $autoRebuild = FALSE, $invalidLinkMode = NULL)
 	{
 		$this->container = $container;
 		$this->autoRebuild = $autoRebuild;
+		$this->invalidLinkMode = $invalidLinkMode;
```

Comment:
I liked more the setter. A lot of people are inheriting the class and this might make an unnecesary BC Break. It would be imho better to add sane default value to the property and create setter.